spec/Knp/Rad/Security/Voter/IsOwnerVoterSpec.php

```diff
@@ -8,22 +8,21 @@
 use Knp\Rad\Security\OwnableInterface;
 use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
 use Symfony\Component\Security\Core\User\UserInterface;
-use Symfony\Component\Security\Core\User\EquatableInterface;
 
 class IsOwnerVoterSpec extends ObjectBehavior
 {
-    function it_is_a_security_voter()
+    public function it_is_a_security_voter()
```

Comment:
Because it's more readable without, and because phpspec itself omits it https://github.com/phpspec/phpspec/blob/master/spec/PhpSpec/Config/OptionsConfigSpec.php#L10